addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,126 @@
+| enableAddressLookup | Switch | If set to ON the coordinates are send to google to lookup a human readable address. Default value is OFF. |
+
+### Device
+
+The following channels are available (if supported by the device):
+
+| Channel ID                   | Type     | Description                                                                                                                                 |
+|------------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| batteryStatus                | String   | Current battery status (Charging, Charged, Unknown)                                                                                         |
+| batteryLevel                 | Number   | Battery charge in %                                                                                                                         |
+| findMyPhone                  | Switch   | Triggers the ["Find my phone"](https://support.apple.com/explore/find-my-iphone-ipad-mac-watch) functionality of the device (if available). |
+| location                     | Location | Location of the device.                                                                                                                     |
+| locationAccuracy             | Number   | Accuracy of the last position report.                                                                                                       |
+| locationLastUpdate           | DateTime | Timestamp of the last location update.                                                                                                      |
+| distanceFromHome             | Number   | Device distance from the location set in openHAB regional settings.                                                                         |
+| addressStreet<sup>*</sup>    | String   | Street component of the address.                                                                                                            |
+| addressCity<sup>*</sup>      | String   | City component of the address.                                                                                                              |
+| addressCountry<sup>*</sup>   | String   | Country component of the addess.                                                                                                            |
+| formattedAddress<sup>*</sup> | String   | Formatted human readable address string.                                                                                                    |
+| deviceName                   | String   | The device name assigned to the device through device settings.                                                                             |
+<sup>*</sup> Address lookup needs to be enabled on the device account bridge thing for this channels to be populated..
```

Comment:
The asterisk irritates GitHub/Markdown. It suspects *italic* formatting. 
Easy solution: Use &#42;: `&#42;` or <sup>i</sup>: `<sup>i</sup>`